config/storage.yml

```diff
@@ -24,7 +24,7 @@ local:
 # Use rails credentials:edit to set the Azure Storage secret (as azure_storage:storage_access_key)
 # microsoft:
 #   service: AzureStorage
-#   storage_account_name: your_account_name
```

Comment:
Not sure if this should be `storage_account_name` in azure, or if we had named that